prefab/curl-ssl/gradle.properties

```diff
@@ -19,4 +19,10 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+
+# enable Prefab feature in gradle plugin 4.0+
```

Comment:
This is actually 4.0 only. 4.1 is moving things from gradle.properties into build.gradle, so this won't work in 4.1.

I'd change this to "Enable importing prefab depenencies".

Reply:
done! this is good one: when 4.1 coming along, this would be updated accordingly